docs/user-guide/cli.md

```diff
@@ -100,6 +100,10 @@ Print the configuration for the given path. Stylelint outputs the configuration
 
 Only register problems for rules with an "error"-level severity (ignore "warning"-level). [More info](options.md#quiet).
 
+### `--quiet-deprecation-warnings`
+
+Ignore deprecation warnings for rules. [More info](options.md#quietDeprecationWarnings)
```

Comment:
[suggest]

- Remove `for rules` because deprecation warnings are not limited to rules (we may add a deprecation warning to a config prop or an CLI flag in the future)
- Convert the hash (#) string to a lower case for consistency
- Add a missing period (.)

```suggestion
Ignore deprecation warnings. [More info](options.md#quietdeprecationwarnings).
```